.github/workflows/release-angular-pr.yml

```diff
@@ -0,0 +1,69 @@
+name: Release Angular on PR
```

Comment:
Should not you put that in edge release ?

Reply:
That's what is made for

Reply:
Nope since we run this on every comments

Reply:
a tag in the commit is not enough ? or publishing it every time like the other packages ?